build.gradle

```diff
@@ -1,31 +1,37 @@
 plugins {
     id 'java'
     id 'idea'
+    id 'io.freefair.lombok' version '5.3.0'
 }
 
+group '${project.group}'
+version '${project.version}'
+generateLombokConfig.enabled = false
+
 repositories {
     maven { url 'https://build.shibboleth.net/nexus/content/groups/public' }
-    jcenter()
```

Comment:
jcenter will be defunk in Feb22 and should be pulled from any gradle build the IAM team hhas used this